gradle-plugin/build.gradle.kts

```diff
@@ -15,11 +15,15 @@ version = "0.1-SNAPSHOT"
 
 repositories {
     mavenCentral()
+    google()
 }
 
 dependencies {
     compile(kotlin("stdlib-jdk8"))
     compile(gradleApi())
+
+    // Ideally, this version would align with the application versions.
+    implementation("com.android.tools.build:gradle:3.2.0")
```

Comment:
afaik, no: this dependency is used to compile the byte code that will be called by the application.

That being said, I realized this probably doesn't matter for the current use case: we're only grabbing the buildVersion from the build file, which is unlikely to change format so if the plugin uses one version of the Android plugin code and the application uses another for the rest of its code, it shouldn't be a problem. For other uses, this could be a problem in the future, but that's unlikely to happen often.